pkg/hints/hints.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2024 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hints
```

Comment:
It's not a general solution to provide `hints` package. You should design a `context` for ingester like Java SDK.

For example,

```
context/
  context.go
```

Reply:
I have now created a context package and moved the code into the context package.